Source/WTF/wtf/SHA1.cpp

```diff
@@ -159,7 +159,7 @@ void SHA1::processBlock()
 {
     ASSERT(m_cursor == 64);
 
-    uint32_t w[80] = { 0 };
+    std::array <uint32_t, 80> w { };
```

Comment:
`{ }` should be unnecessary.

Reply:
I think it _is_ necessary. Otherwise the elements aren't value-initialized to 0.

Reply:
You are right. I missed the fact that we wanted to zero-initialize.